integration/smoke/run_smoke.sh

```diff
@@ -31,8 +31,22 @@
 #  OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #
 
-source geopm_env.sh
-EXP_DIR=$GEOPM_SRC/integration/experiment
+
+# TODO: fix this
+if [ -f ${HOME}/.geopmrc ]; then
+    source ${HOME}/.geopmrc
+fi
+
+if [ ! -z ${GEOPM_SYSTEM_ENV} ]; then
+    source ${GEOPM_SYSTEM_ENV}
+fi
+
+GEOPM_SOURCE=${GEOPM_SOURCE:?Please set GEOPM_SOURCE in your environment.}
+source ${GEOPM_SOURCE}/integration/config/run_env.sh
+EXP_DIR=${GEOPM_SOURCE}/integration/experiment
+
+APPLICATIONS="dgemm dgemm_tiny nekbone minife amg nasft hpcg hpl_mkl hpl_netlib pennant"
+
```

Comment:
If the lines here are identical to gen_smoke.sh lines 34-48, I think it's a good candidate for an extract refactor.  I think it would be easy to get the files out-of-sync in these sections otherwise.

Perhaps it makes sense to put this all in new ```smoke_env.sh``` in this dir and source it here?

Reply:
that makes sense to me.  That may have been what "fix this" was about but I really can't remember